### PR TITLE
fix: group enable/disable endpoints need url param

### PR DESCRIPTION
**Pull request by caproven**
Quick fix for receiver group enable/disable endpoints, which don't seem to currently work. I originally tried using query params (ex. `api/v1/groups/enable?groupID=myid`) to no avail. It sems `chi.urlParam()` only fetches params provided in routing such as `r.Route("/{param}", handler)`.

With change, endpoints are hit with
- PUT `api/v1/groups/GROUP_ID/enable`
- PUT `api/v1/groups/GROUP_ID/disable`

### CODE REVIEW

**Review by xbcsmith — DISMISSED** [review]

**Comment by xbcsmith** [comment]
Need to update the doc to reflect this feature.

**Comment by gbuskey** [comment]
im pretty sure this is happening because our import are messed up. can you try reverting your change and updating our chi import paths to use v5?

**Review by ninjamast3r — APPROVED** [review]

**Comment by caproven** [comment]
> im pretty sure this is happening because our import are messed up. can you try reverting your change and updating our chi import paths to use v5?

I tried this and it didn't work - unless I changed the handler this was the only way I could get things working.

> Need to update the doc to reflect this feature.

Wasn't sure where exactly to doc this. Added a piece in the glossary, please see if that suffices for now.

**Comment by xbcsmith** [comment]
> > im pretty sure this is happening because our import are messed up. can you try reverting your change and updating our chi import paths to use v5?
> 
> I tried this and it didn't work - unless I changed the handler this was the only way I could get things working.
> 
> > Need to update the doc to reflect this feature.
> 
> Wasn't sure where exactly to doc this. Added a piece in the glossary, please see if that suffices for now.

Doc goes in reference. Probably should start an api.md or something along those lines

**Review by xbcsmith — CHANGES_REQUESTED** [review]
We should change this to a PATCH on /groups/{ID} to enable/disable

File: pkg/api/api.go [hunk 1]
```diff
@@ -90,10 +90,10 @@ func Initialize(db *storage.Database, msgProducer message.TopicProducer, cfg *co
 			})
 			r.Route("/groups", func(r chi.Router) {
 				r.Post("/", s.Rest.CreateGroup())
-				r.Put("/enable", s.Rest.SetGroupEnabled(true))
-				r.Put("/disable", s.Rest.SetGroupEnabled(false))
 				r.Route("/{groupID}", func(r chi.Router) {
 					r.Get("/", s.Rest.GetGroupByID())
+					r.Put("/enable", s.Rest.SetGroupEnabled(true))
```
Comment (ninjamast3r) [inline pkg/api/api.go:95]: The team talked about this on our Friday call. We think the better solution is is to do a PATCH request to the /{groupID} with the enabled field changed. After thinking about it, that seemed like the more REST-y solution.

**Review by ninjamast3r — CHANGES_REQUESTED** [review]

File: pkg/api/api.go (hunk 1 above)
Comment (caproven) [reply pkg/api/api.go]: Switched to PATCH `/groups/{groupID}`, please take a look at implementation

**Review by ninjamast3r — COMMENTED** [review]
Looks ok to me. Rebase, and I'll approve it.

**Review by xbcsmith — APPROVED** [review]

**Review by xbcsmith — APPROVED** [review]

**Pull request merged**